dotnet/src/IntegrationTests/Plugins/RepairServiceTests.cs

```diff
@@ -126,4 +180,22 @@ public class Repair
         [JsonPropertyName("image")]
         public string? Image { get; set; }
     }
+
+    public sealed class CustomHandler(HttpMessageHandler innerHandler) : DelegatingHandler(innerHandler)
```

Comment:
If this CustomHandler is used only in the scope of this TestClass, suggest setting as `private`

```suggestion
    private sealed class CustomHandler(HttpMessageHandler innerHandler) : DelegatingHandler(innerHandler)
```